library/src/main/java/com/pokegoapi/api/pokemon/EvolutionInfo.java

```diff
@@ -419,6 +419,24 @@
 	}
 
 	/**
+	 * Get evolutions
+	 *
+	 * @param pokemonId pokemon id
+	 * @return ordered evolutions
+	 */
+	public static List<PokemonId> getEvolutions(PokemonId pokemonId) {
```

Comment:
Why you not just used new ArrayList<PokemonId>(Arrays.asList(array)) to convert the Array to a ArrayList.